src/System.Drawing.Common/tests/Drawing2D/MatrixTests.cs

```diff
@@ -375,20 +374,24 @@ public void Multiply_InvalidMatrixOrder_ThrowsArgumentException(MatrixOrder orde
         [ConditionalFact(Helpers.IsDrawingSupported)]
         public void Multiply_Disposed_ThrowsArgumentException()
         {
+            var disposedMatrix = CreateDisposedMatrix();
```

Comment:
Nit: var => Matrix, here and in similar cases below

Reply:
Thanks, should be OK now (and similar for Region)